.github/scripts/generate_binary_build_matrix.py

```diff
@@ -248,7 +248,8 @@ def generate_wheels_matrix(
                         "nvidia-cusolver-cu12==11.4.5.107; platform_system == 'Linux' and platform_machine == 'x86_64' | "
                         "nvidia-cusparse-cu12==12.1.0.106; platform_system == 'Linux' and platform_machine == 'x86_64' | "
                         "nvidia-nccl-cu12==2.18.1; platform_system == 'Linux' and platform_machine == 'x86_64' | "
-                        "nvidia-nvtx-cu12==12.1.105; platform_system == 'Linux' and platform_machine == 'x86_64'",
+                        "nvidia-nvtx-cu12==12.1.105; platform_system == 'Linux' and platform_machine == 'x86_64' | "
+                        "triton==2.1.0; platform_system == 'Linux' and platform_machine == 'x86_64'",
```

Comment:
I'm trying to understand why we set `triton` here instead of `pytorch-triton` package that we build.

Reply:
triton is package from pypi